### ISSUE TITLE
Powerless Pearl (Witching Gadgets) Dupe Glitch

### ISSUE THREAD

**Issue by xSable** [makeoo/Gadomancy#64]
NOTICE: WILL ALSO POST THIS IN WITCHING GADGETS ISSUES!!!

So here's the deal:

This bug occurs when using Gadomancy(Gado) and Witching Gadgets(WG).

Specifically when making Primordial Armors (WG) and using Disguise Infusion (Gado). When you first craft the armors, it uses 3 primordial pearls, but when the infusion is finished it drops the 3 powerless pearls, so you can recharge them.

Now, once you've made them, let's say you want to disguise them.

You go through the infusion, and bam, 3 more powerless pearls drop. Now, just recharge them all, and you've got 6 pearls when you started with 3.

That's definitely a problem.

**Comment by Shadows-of-Fire** [comment]
Yeah well, being WG, it's not going to get fixed on their end.  You probably should just drop that mod unless it returns to development.  I don't think its possible to make the powerless pearl not drop anyway, since what sounds like is happening is that it's calling an event on infusion complete for the itemID for the armors.

**Comment by HellFirePvP** [comment]
Could be because WG has some special handling regarding Infusions that leave powerless perals behind. Infusions don't do that normally so i can see WG might've done something tricky to make it happen. So using it in a disguise infusion could cause WG to believe that it's the armor's infusion not a disguise infusion. So yea. Pretty sure it's on WG's end. We will not fix bugs in WG.
